src/data_provider/src/packages/rpmlibWrapper.h

```diff
@@ -0,0 +1,30 @@
+#ifndef RPMLIB_WRAPPER_H
```

Comment:
Add license comment block
```suggestion
/* Copyright (C) 2015-2021, Wazuh Inc.
 * All rights reserved.
 *
 * This program is free software; you can redistribute it
 * and/or modify it under the terms of the GNU General Public
 * License (version 2) as published by the FSF - Free Software
 * Foundation
 */
 
#ifndef RPMLIB_WRAPPER_H
```